mods/tuxemon/db/npc/spyder_barmaid.json

```diff
@@ -0,0 +1,4 @@
+{
+    "slug": "spyder_barmaid",
```

Comment:
npcs now require combat_front and combat_back sprites to be defined.

Reply:
Thanks - will fix.